scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -450,4 +450,30 @@ mod tests {
             .await
             .unwrap();
     }
+
+    #[tokio::test]
+    async fn test_partition_delete() {
+        let schema = TestTableSchema {
+            name: "PARTITION_DELETE".to_string(),
+            partition_key: vec![("pk", "int")],
+            clustering_key: vec![("ck", "int")],
+            other_columns: vec![("v", "int")],
+        };
+
+        let operations = vec![
+            (
+                "INSERT INTO PARTITION_DELETE (pk, ck, v) VALUES (?, ?, ?)",
+                vec![Int(0), Int(0), Int(0)],
+            ),
+            (
+                "INSERT INTO PARTITION_DELETE (pk, ck, v) VALUES (?, ?, ?)",
+                vec![Int(0), Int(1), Int(1)],
+            ),
+            ("DELETE FROM PARTITION_DELETE WHERE pk = ?", vec![Int(0)]),
```

Comment:
Try a test where the partition key column is named differently than `pk`, or where there are multiple partition key columns.